robottelo/cli/gpgkey.py

```diff
@@ -38,25 +38,23 @@ def info(cls, options=None):
         cls.command_sub = 'info'
 
         result = cls.execute(
-            cls._construct_command(options), output_format='csv')
+            cls._construct_command(options), output_format='json')
+
+        if isinstance(result, dict) or len(result) == 0:
```

Comment:
This seems like a bug, how is it possible to get a list when a single info dict is expected? That is weird.

To me info should always return a single dict with the information about the entity. On the other hand a list should always return a list of dicts with information.